backend/README.md

```diff
@@ -1,8 +1,49 @@
 # Backend For FreshNest
-> Node.js
+Prerequisites: 
```

Comment:
I would suggest you make the Prerequisites a subheading an `h2`, we can then have a similar **Steps** heading before the steps start.

Eg:
## Prerequisites:
## Steps